tests/e2e/set-ffi-env-e2e

```diff
@@ -273,12 +242,27 @@ then
     exit
 fi
 
+if stat /run/ostree-booted > /dev/null 2>&1; then
+   info_message "Warning: script can not run on ostree image"
+   info_message "=============================="
+   exit 0
+fi
+
+# Creates  partitions for QM env on VM
+echo
+info_message "Check if qm requires additional partition"
+info_message "=============================="
+if [ -n "${SET_QM_PART}" ]; then
+    create_qm_var_part
+fi
+
 echo
 info_message "Checking if QM already installed"
 info_message "=============================="
 QM_STATUS="$(systemctl is-enabled qm 2>&1)"
 if [ "$QM_STATUS" == "generated" ]; then
    if [ "$(systemctl is-active qm)" == "active" ]; then
+       systemctl restart qm
```

Comment:
This is interesting: if it's active why restart?

Reply:
doing restart for the new /var partition added earlier.
I can check for disk partition